src/pages-and-resources/live/data/thunks.js

```diff
@@ -0,0 +1,110 @@
+import { history } from '@edx/frontend-platform';
+import { getLiveConfiguration, getLiveProviders, postLiveConfiguration } from './api';
+import {
+  DENIED, FAILED, LOADING, LOADED, SAVING, SAVED, updateStatus, updateSaveStatus,
```

Comment:
if you follow the above convention you will save import lines here and you will just have to import REQUEST_STATUS.